AerospikeClient/Cluster/PartitionParser.cs

```diff
@@ -192,8 +194,12 @@ private void DecodeBitmap(Node node, Partitions partitions, int index, int regim
 		{
 			Node[] nodeArray = partitions.replicas[index];
 			int[] regimes = partitions.regimes;
-			char[] chars = Encoding.ASCII.GetChars(buffer, begin, offset - begin);
-			byte[] restoreBuffer = Convert.FromBase64CharArray(chars, 0, chars.Length);
+
+			Span<byte> bufferChars = buffer.AsSpan(start: begin, length: offset - begin);
+			Span<byte> restoreBuffer = stackalloc byte[Base64.GetMaxDecodedFromUtf8Length(bufferChars.Length)];
+			var decodeStatus = Base64.DecodeFromUtf8(bufferChars, restoreBuffer, out _, out int restoreBufferLength);
+			Debug.Assert(decodeStatus == OperationStatus.Done);
```

Comment:
Remove debug statement.

Reply:
Note that this debug statement is only executed for debug build and will not be executed in production code. I would prefer to get a very early error if the code is modified in a way that would make the decoding fail.

Reply:
The newline is expected, it is part of our wire protocol.